src/Umbraco.Cms.Integrations.Commerce.Shopify/build/Umbraco.Cms.Integrations.Commerce.Shopify.targets

```diff
@@ -4,9 +4,10 @@
 		<ShopifyPropertyEditorFiles>$(MSBuildThisFileDirectory)..\App_Plugins\UmbracoCms.Integrations\Commerce\Shopify\**\*.*</ShopifyPropertyEditorFiles>
 	</PropertyGroup>
 
-	<Target Name="CopyShopifyPropertyEditorAssets" BeforeTargets="Build">
+	<Target Condition=" '$(TargetFramework)' != 'net472' " Name="CopyShopifyPropertyEditorAssets" BeforeTargets="Build">
```

Comment:
Shouldn't this condition be on the `ShopifyPropertyEditorFiles` item group instead, since this target needs to run on both .NET Framework and modern .NET?

And instead of adding multiple includes/excludes, use this instead:
```
<ShopifyPropertyEditorFiles Include="$(ShopifyPropertyEditorFiles)" />
<ShopifyPropertyEditorFiles Condition="'$(TargetFramework)' != 'net472'" Remove="$(MSBuildThisFileDirectory)..\App_Plugins\UmbracoCms.Integrations\Commerce\Shopify\RenderV8\*.*" />
<ShopifyPropertyEditorFiles Condition="'$(TargetFramework)' == 'net472'" Remove="$(MSBuildThisFileDirectory)..\App_Plugins\UmbracoCms.Integrations\Commerce\Shopify\RenderV9\*.*" />
```

Reply:
Thanks for the suggestion @ronaldbarendse , updated accordingly. On V8 it works fine.